taskcluster/taskcluster
# [UI] Bug-1306494 Add diff view

Bugzilla: [1306494](https://bugzilla.mozilla.org/show_bug.cgi?id=1306494)

### Description
* Use [react-gh-like-diff](https://github.com/neighborhood999/react-gh-like-diff) to implement `<DiffTextArea />` component to render `diff view`
* I refer the style refer to [diff2html](https://github.com/rtfpessoa/diff2html/blob/master/src/ui/css/diff2html.css) and do some adjustment

### How to test
1.  Go to http://localhost:5080/auth/clients/email%2Fbastien%40nextcairn.com%2Ftc-notify-test
2. Update description in `Scope`
3. Click `VIEW DIFF` button
4. See the diff view display
